src/Admin/CategoryAdmin.php

```diff
@@ -68,7 +68,7 @@ public function getFormBuilder()
     protected function configureFormFields(FormMapper $formMapper)
     {
         $formMapper
-            ->with('General', ['class' => 'col-md-6'])
+            ->with('label_general', ['class' => 'col-md-6'])
```

Comment:
`label_` is very generic. Maybe use `group_` as a better prefix

Reply:
done